### ISSUE TITLE
The accuracy rate on the test set is not ideal, how to solve it ？

### ISSUE THREAD

**Issue by chen-love** [Aeryes/Kaggle-Flower-Recognition-CNN#10]
I have slightly modified your code to divide the training and testing sets at 8:2. The accuracy rate on the training set can reach 99%, but the testing set can only reach 66%

**Comment by Aeryes** [comment]
Sounds like overfitting.

There are a few things you can try. 
1. Data augmentation.
2. Dropout.
3. Print graphs of the accuracy for train and validation and see where it converges. Lower epochs closer to convergence.
4. Increase the size of the data set.
5. Adjust the learning rate.
6. Use weight decay.
